core/arch/arm/kernel/thread_spmc.c

```diff
@@ -748,7 +748,8 @@ static void handle_blocking_call(struct thread_smc_args *args,
 	case OPTEE_FFA_GET_OS_VERSION:
 		spmc_set_args(args, direct_resp_fid, swap_src_dst(args->a1), 0,
 			      CFG_OPTEE_REVISION_MAJOR,
-			      CFG_OPTEE_REVISION_MINOR, TEE_IMPL_GIT_SHA1);
+			      CFG_OPTEE_REVISION_MINOR,
+			      SHIFT_U32(TEE_IMPL_GIT_SHA1, 16));
```

Comment:
What is this supposed to do?

Reply:
Oops. I want to keep the upmost 32 bits. Clearly not what this is doing :cry: `TEE_IMPL_GIT_SHA1 >> 16` maybe? :wink: